keyboards/ergodox_ez/keymaps/default/keymap.c

```diff
@@ -194,6 +194,8 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
 
 // Runs just one time when the keyboard initializes.
 void matrix_init_user(void) {
+  rgblight_init();
```

Comment:
Not needed.  This happens regardless, as long as the RGBLights are enabled.

Reply:
Also, now we have `keyboard_post_init_user` that happens after rgblight initialization occurs